.github/workflows/checks.yml

```diff
@@ -86,13 +95,33 @@ jobs:
         make attribute-registry-generation
         git diff --exit-code './docs/attributes-registry/*.md' || (echo 'Attribute registry markdown is out of date, please run "make attribute-registry-generation" and commit the changes in this PR.' && exit 1)
 
+  semantic-conventions-registry-podman:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: install podman and podman-docker
+      run: sudo apt-get install podman podman-docker
+    - name: verify registry tables
+      run: |
+        make attribute-registry-generation
+        git diff --exit-code './docs/attributes-registry/*.md' || (echo 'Attribute registry markdown is out of date, please run "make attribute-registry-generation" and commit the changes in this PR.' && exit 1)
```

Comment:
what do you think of using a matrix build (e.g. podman `false`/`true`) to avoid the worry of keeping these jobs synchronized?

Reply:
Done. Switched to using the matrix mechanism. Thanks for the tip.